doofinder-for-woocommerce/includes/api/class-store-api.php

```diff
@@ -88,13 +88,12 @@ public function create_store($api_keys)
      * $payload = array(
      *    'store_options' => array(
      *        'url' => 'http://pedro-wordpress.ngrok.doofinder.com',
-     *        'api_pass' => 'G41cXNeVoX4JGL2bhvbcMlQ4',
-     *        'api_user' => 'pedro'
+     *        'df_token' => 'G41cXNeVoX4JGL2bhvbcMlQ4'
      *    ),
      *    'search_engines' => array(
      *        'fde92a8f364b8d769262974e95d82dba' => array(
      *          'feed_type' => 'post',
-     *          'url' => 'http://pedro-wordpress.ngrok.doofinder.com'
+     *          'lang' => 'http://pedro-wordpress.ngrok.doofinder.com'
```

Comment:
If it is currently `lang` and not `url`, I think this example might be wrong.

Reply:
outdate!